### ISSUE TITLE
Antiforgery filter is in wrong order relative to Authorize

### ISSUE THREAD

**Issue by Eilon** [aspnet/Mvc#5483]
See original issue w/ discussion and analysis here: https://github.com/aspnet/Security/issues/1009

And pending PR here: https://github.com/aspnet/Mvc/pull/5470

**Comment by Eilon** [comment]
This patch is **approved**, please ensure it is merged into the correct branch and building as part of the patch train.

**Comment by Eilon** [comment]
@ryanbrandenburg please complete your PR, check into the patch build, then port to `dev` to ensure it goes into the 1.1.0 release. I logged https://github.com/aspnet/Mvc/issues/5484 to track the 1.1.0 work to make sure we don't forget.

**Comment by ryanbrandenburg** [comment]
Fixed by #5470.

**Comment by ryanbrandenburg** [comment]
Currently waiting on approval of #5470.

**Comment by ryanbrandenburg** [comment]
Merged in f02f55c. Still waiting on the PR to add rel/1.0.2 to the patch train.

**Comment by ryanbrandenburg** [comment]
rel/1.0.2 is in the patch train now, closing this.
